libraries/freertos_plus/aws/ota/src/aws_ota_cbor.c

```diff
@@ -37,7 +37,7 @@
  * @brief Message field definitions, per the server specification.
  */
 
-#define OTA_CBOR_GETSTREAMREQUEST_ITEM_COUNT    5
+#define OTA_CBOR_GETSTREAMREQUEST_ITEM_COUNT    6
```

Comment:
Could that parameter automatically scale?

Reply:
Unfortunately, the tiny cbor library needs to specify number of parameters upfront before creating the map. If needed otherwise we might need to use the non-finite length maps, which require changes in server side I believe.